src/utils.ts

```diff
@@ -0,0 +1,37 @@
+export function u8ArrayToBytes(array: Uint8Array) {
+  let ret = "";
+  for (let e of array) {
+    ret += String.fromCharCode(e);
+  }
+  return ret;
+}
+
+// TODO this function is a bit broken and the type can't be string
+// TODO for more info: https://github.com/near/near-sdk-js/issues/78
+export function bytesToU8Array(bytes: any): Uint8Array {
+  let ret = new Uint8Array(bytes.length);
+  for (let i in bytes) {
+    ret[i] = bytes.charCodeAt(i);
+  }
+  return ret;
+}
+
+export function bytes(strOrU8Array: string | Uint8Array): string {
```

Comment:
And this would be `export function bytes(strOrU8Array: string | Uint8Array): Bytes {`